assets/section-main-product.css

```diff
@@ -248,6 +248,12 @@ a.product__text {
   padding: 0 1rem;
 }
 
+.product__accordion .accordion__content:after {
+  clear: both;
+  content: "";
+  display: block;
+}
```

Comment:
I think you're right in your description. It would need to be added for the `.rte` class I think. Cause if you go on the home page or anywhere else you can add a section, then add a `page` section and use `page with floated image` then you will get the same issue: 
![](https://screenshot.click/21-07-cr28x-jakn0.png)

Reply:
True, that will come up more often then. Moved the same fix to the rte class now. I can't think of a reason why this wouldn't be ok, but maybe there's a rte case out there I'm not thinking of. Curious if anyone thinks there's a safer way to handle this.

Reply:
I guess we could have a `rte-page` specific class but I don't think having it to the more general `rte` one should create any problems. It's adding content in places where it might not be needed but not sure if we should avoid that or not. We can check with the second reviewer.

Reply:
I think it should be fine.

Reply:
Also I haven't seen a clearfix in a long time. This brings me back!

Reply:
Yeah I'm ok with the content as is. I'll leave it simple as is if you don't have a strong objection Ludo